projects/lib/src/admin/components/page-list/page-list.component.ts

```diff
@@ -58,8 +59,12 @@ export class PageListComponent implements OnInit {
         'cms-page-list missing required props resourceType and resourceID for '
       );
     }
-    return HeadStartSDK.Documents.ListDocuments(PAGE_SCHEMA.ID, this.resourceType, this.resourceID)
-      .then((response) => this.list = response.Items)
+    return HeadStartSDK.Documents.ListDocuments(
```

Comment:
I don't think we have to change this now but I think we might want to follow the pattern for Assets to keep it consistent